core/src/main/scala/Json.scala

```diff
@@ -505,12 +509,22 @@ object Json {
       e <- (z --\ "repository" --\ "id").as[Long]
     } yield {
       Github.ReleaseEvent(
-        id = a,
+        id = a.toString,
```

Comment:
Why not decode it as a `String` above, instead of calling `toString` here?

Reply:
Sure, take it for granted.